packages/enzyme-matchers/src/utils/displayName.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+export default function displayName(Component: Function): string {
```

Comment:
instead of copypastaing this, why not import https://unpkg.com/airbnb-prop-types/build/helpers/getComponentName.js ?

Reply:
That seems reasonable to me, but it looks like airbnb-prop-types would be a new dependency here. Any objections to adding a dependency on airbnb-prop-types?

Reply:
It's not my project :-) but i'm not sure why adding a dep to a test framework would be a problem; especially considering airbnb-prop-types and enyzme are both maintained by me, and i'm unlikely to do anything that breaks the enzyme ecosystem :-)

Reply:
I'm only thinking about the stability of this project that i like having this logic here. if anything changes with React, I don't want one of us to be stuck waiting for other to do updates properly to support. This code is small and uncomplex, so this is fine.

My only request here is to get the flow type correct. You're defining it as a Function, but the first line is checking if it's a string. Should this be `React.Node | string` ?

Reply:
@blainekasten if airbnb-prop-types doesn't update, then enzyme itself would be "broken" in that case - and I'd suggest that perhaps enzyme-matchers should always match enzyme, which means using the same implementation.

Reply:
> My only request here is to get the flow type correct. You're defining it as a Function, but the first line is checking if it's a string. Should this be React.Node | string ?

Nice catch! Updated.